src/app/inventory/d2-stores.ts

```diff
@@ -413,7 +414,11 @@ function makeD2StoresService(): D2StoreServiceType {
   function updateBasePower(stores: D2Store[], store: D2Store, defs: D2ManifestDefinitions) {
```

Comment:
There was a branch where I tried to remove all the stats from `DimStore` and just compute them on the fly in the display component from items so we didn't have to ping the API every time you equip an item, and then I realized that classified items exist :-(